CHANGES.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 1.12.2 (core) / 0.28.2 (libraries)
+
+### New
+
+- A new sidebar in the Asset catalog (behind a feature flag) provides a hierarchical view of your asset keys.
+- Added `dg api schedule {list,get}` command.
+- [dagster-census] Added `CensusComponent` allowing Dagster assets to sync with census connections.
+- [ui] Improve a few parts of the "Recent updates" timeline section of the Asset overview page.
```

Comment:
we should sort these as

- no tag
- [ui]
- [some integration]

(same goes for the section below)